external/CapnProto.cmake

```diff
@@ -35,7 +35,7 @@ endif ()
 if (NOT CAPNP_FOUND)
   # Build Cap'n Proto from source.
   if(${CMAKE_SYSTEM_NAME} MATCHES "Windows")
-    set(CAPNP_DEFINITIONS "-DCAPNP_LITE=1")
+    set(CAPNP_DEFINITIONS "-DCAPNP_LITE=1 -DEXTERNAL_CAPNP=1")
```

Comment:
Oh this PR helps me understand this. `EXTERNAL_CAPNP` controls whether you get the command line tools separately?